cypress/e2e/michael.cy.ts

```diff
@@ -0,0 +1,16 @@
+describe('drag', () => {
+    xit('click twice', () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/72954');
+        cy.get('.btn').click().click()
```

Comment:
It is not safe to chain after click(). The subject is very likely to change after click() action. So it is better to find the element again. https://obstaclecourse.tricentis.com/Obstacles/72954

btw, '.btn' is not a good locator, as it is too general and easy to break.